lib/cartopy/tests/mpl/test_axes.py

```diff
@@ -119,3 +120,15 @@ def test_styler_kwarg(self, ShapelyFeature, add_feature_method):
 def test_geoaxes_subplot():
     ax = plt.subplot(1, 1, 1, projection=ccrs.PlateCarree())
     assert str(ax.__class__) == "<class 'cartopy.mpl.geoaxes.GeoAxesSubplot'>"
+
+
+@ImageTesting(['geoaxes_subslice'], tolerance=0)
```

Comment:
Default is 0.5; looks like you might need that. I'm sure the missing line would be >0.5.

Reply:
I didn't realize that was the default. Yeah, I think it was an RMS of ~6 when the test failed for me locally.